### DISCUSSION TITLE
No faces are detected

### DISCUSSION THREAD

**Discussion by kpine** [photoview/photoview#342] [Q&A and Troubleshooting] [answered]
Using docker image tag `viktorstrate/photoview:2`, I setup a new instance of photoview. However, after scanning all photos the People page is completely blank. I have a mix of jpg and raw files. Did I miss something that is required to enable face detection? I don't see anything in the scanning logs to indicate either errors or success.

**Comment by viktorstrate** [comment]
Faces are only scanned the first time an image is scanned, if you have updated from a version without face recognition, you'll have to remove your media and completeley rescan them, there are unfortunateley no better way to do this at the moment.

To remove the images from the database, all references to it for any user in the Settings page, then add the paths back in and rescan.

**Reply by mungo312** [reply]
I've the same problem as the thread starter. I setup a new instance of phtotoview, no migration etc. and there are no faces detected and no hints or errors in the logs

**Reply by viktorstrate** [reply]
Oh, this is a bug.
I'll make sure to fix this before next release.

**Comment by kpine** [comment, marked as answer]
Fixed in 2.3.2, thanks!
